Scheduler/v3/fbs_tts_20211007_spec_image_survey.yaml

```diff
@@ -0,0 +1,83 @@
+driver_type: lsst.ts.scheduler.driver.feature_scheduler
+mode: ADVANCE
+s3instance: tuc
+models:
+  observatory_model:
+    camera:
+      filter_max_changes_burst_num: 1
+      filter_max_changes_avg_num: 30000
+    optics_loop_corr:
+      tel_optics_cl_alt_limit:
+        - 0
+        - 30
+        - 90
+driver_configuration:
+  default_observing_script_name: auxtel/track_target_and_take_image.py
+  default_observing_script_is_standard: true
+  stop_tracking_observing_script_name: auxtel/stop_tracking.py
+  scheduler_config: ts_config_ocs/Scheduler/feature_scheduler/auxtel/2021/10/07/fbs_config_spec_image_survey.py
+  survey_observing_script:
+    cwfs:
+      observing_script_name: auxtel/latiss_cwfs_align.py
+      observing_script_is_standard: false
+    spec:
+      observing_script_name: auxtel/latiss_acquire_and_take_sequence.py
+      observing_script_is_standard: false
+  script_configuration:
+    script_configuration:
+      grating: empty_1
+      band_filter: SDSSg
+    script_configuration_cwfs:
+      filter: FELH0600
+      find_target:
+        mag_limit: 8.0
+    script_configuration_spec:
+      do_acquire: True
+      acq_filter: BG40
+      acq_grating: ronchi170lpmm
+      do_take_sequence: True
+      exposure_time_sequence:
+        - 20
+        - 20
+        - 20
+        - 20
+        - 20
+        - 20
+        - 20
+        - 20
+      filter_sequence:
+        - empty_1
+        - SDSSg
+        - BG40
+        - FELH0600
+        - empty_1
+        - SDSSg
+        - BG40
+        - FELH0600
+      grating_sequence:
+        - ronchi170lpmm
+        - ronchi170lpmm
+        - ronchi170lpmm
+        - ronchi170lpmm
+        - holo4_003
+        - holo4_003
+        - holo4_003
+        - holo4_003
+telemetry:
+  efd_name: ncsa_teststand_efd
```

Comment:
This isn't going to work on TTS. Need tucson_teststand_efd.

Reply:
cool! thanks for catching this one!